qa/workunits/rbd/rbd_mirror_deletion.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/sh -ex
+#
+# rbd_mirror_deletion.sh - test rbd-mirror daemon with image deletion
```

Comment:
Wouldn't it be better to modify the existing tests (rbd_mirror_{journal,snapshot,stress}.sh)? The cluster setup is rather an expensive step, and I am not sure it is useful to make many small tests.

Any you would also need to add a yaml(s) to qa/suites/rbd to be able to test this on teuthology.

Reply:
Oh right ok, I will add this to existing tests. I just need to make sure that all the images are removed "manually" before doing this test on OMAPs, may need some further tweaking